src/test/scala/circtTests/stage/ChiselStageSpec.scala

```diff
@@ -334,6 +364,60 @@ class ChiselStageSpec extends AnyFunSpec with Matchers with chiselTests.Utils {
 
       os.read(targetDir / "HasUnserializableAnnotation.fir") shouldNot include("DummyAnnotation")
     }
+
+    it("should allow building a module from JSON") {
+      import ChiselStageSpec.{Corge, Parameters}
+
+      val targetDir = baseDir / "should-allow-building-a-module-from-JSON"
+
+      val clazz = classOf[Corge].getName()
+      val json = ChiselGeneratorAnnotation.serializeJSON(Parameters(42), classOf[Parameters] :: Nil)
```

Comment:
what is in this json file? Would it be worth having this test show it / write one out explicitly? is the JSON supposed to stand on its own or only make sense if we call it from something ? What is the relevance of evaluating `clazz` could I just make up a string here and everything just works?

Reply:
This is the exact output:

```json
{"class":"circtTests.stage.ChiselStageSpec$Parameters","width":42}
```

The API is `--module-json <class-name-string>,<json-string>`. `clazz` isn't being evaluated. I'm programmatically calling the `getName` method in order to not hard-code `chiselTests.whatever.Corge`.